tests/utils.py

```diff
@@ -582,5 +583,24 @@ def query_memory(n=2):
     return ",".join([f"{int(x)}" for x in ids])
 
 
+def download_if_not_exist(file_url, file_path=None):
```

Comment:
could use https://github.com/Project-MONAI/MONAI/blob/master/monai/apps/utils.py#L74 we could have a hash for the file to detect external changes.  there's an example usage here: https://github.com/Project-MONAI/MONAI/blob/master/tests/clang_format_utils.py#L60